tools/genevergreen/generate/generate.go

```diff
@@ -215,7 +216,7 @@ func PublishSnapshotTasks(c *shrub.Configuration) {
 	publishVariant(
 		c,
 		v,
-		"5.0",
+		"8.0",
```

Comment:
FYI snapshots don't go to a the repo here they should go to [testing](https://repo.mongodb.org/apt/ubuntu/dists/focal/mongodb-org/testing) but that's based on the barque key we use